packages/block-library/src/template-part/index.php

```diff
@@ -65,8 +65,12 @@ function render_block_core_template_part( $attributes ) {
 		}
 	}
 
-	if ( ! $template_part_id || is_null( $content ) ) {
-		return __( 'Template Part not found.' );
+	if ( is_null( $content ) && is_user_logged_in() ) {
```

Comment:
Should this check for a user role and not only logged in users?

Reply:
I was thinking about that too... Maybe `current_user_can( edit_themes );` 🤔

Reply:
Perhaps leave it as is until https://github.com/WordPress/gutenberg/issues/27597

Reply:
^ That's why I approved the PR 😆